### ISSUE TITLE
sweeper/street_types.json

### ISSUE THREAD

**Issue by rkelson** [agrc/sweeper#44]
The file https://github.com/agrc/sweeper/blob/master/src/sweeper/street_types.json should probably be updated to use the one more specific to utah and be similar to the AGRC_Street_Suffixes.csv. I emailed the file to Scott and Steve.

**Comment by stdavis** [comment]
@rkelson: What specifically are you asking to be updated? I compared the two datasets. There are no missing street types that are in the CSV but not in the new JSON file. 

I found that these types are not in the old CSV but are in the new JSON file: `'ALY', 'BAY', 'BND', 'COR', 'CRES', 'CRK', 'CTR', 'CYN', 'EST', 'ESTS', 'FLT', 'FRK', 'GLN', 'GRV', 'GTWY', 'HL', 'HOLW', 'HTS', 'JCT', 'LNDG', 'MDW', 'MDWS', 'MNR', 'PARK', 'PASS', 'PATH', 'PLZ', 'PT', 'RDG', 'RNCH', 'RTE', 'RUN', 'SPUR', 'SQ', 'TER', 'TRCE', 'VIS', 'VLG', 'VW'`

Are these new types causing issues for you? I know that you mentioned `FRK` in #43, but I did find that there are addresses with that street type.

Or are you saying that some of the commonly used values in the new JSON are incorrect or missing?
